src/components/Code/index.jsx

```diff
@@ -11,6 +12,11 @@ const validLanguage = (props, propName) => {
     return new Error(`Language '${language}' not supported by highlight.js`);
   }
 };
+@withStyles({
+  textCode: {
+    color: 'black',
```

Comment:
Do you know why this is needed now?

Reply:
After migration we've got new css rule in body

![css rule](https://user-images.githubusercontent.com/44370635/67888078-b00f0d80-fb76-11e9-907f-519fb0aeaf15.PNG)

Color is inherited from it.
I think this solution is not good, if `code` will be added to "dark" site, color will stay black I guess.
What can we do here?

Reply:
Perfect! thanks! 👍